front/src/config/config.dev.json

```diff
@@ -1,3 +1,4 @@
 {
-  "serverPath": "https://localhost:5001/"
+  "serverPath": "https://localhost:5001/",
+  "emailRegex": "^\\w+([\\.-]?\\w+)*@\\w+([\\.-]?\\w+)*(\\.\\w{2,3})+$"
```

Comment:
why here? actually this file is not for those purposes

Reply:
Should I create a config file for validation rules ?

Reply:
it is not config. if you want you can create file with constants